include/grpcpp/support/proto_buffer_writer.h

```diff
@@ -149,6 +151,37 @@ class ProtoBufferWriter : public grpc::protobuf::io::ZeroCopyOutputStream {
   /// Returns the total number of bytes written since this object was created.
   int64_t ByteCount() const override { return byte_count_; }
 
+#ifdef GRPC_PROTOBUF_CORD_SUPPORT_ENABLED
+  // Writes cord to the backing byte_buffer, sharing the memory between the
+  // blocks of the cord, and the slices of the byte_buffer.
+  virtual bool WriteCord(const absl::Cord& cord) {
+    grpc_slice_buffer* buffer = slice_buffer();
+    size_t cur = 0;
+    for (absl::string_view chunk : cord.Chunks()) {
+      if (chunk.size() < 512) {
```

Comment:
From our discussion, I don't understand where the `512` number came from. Sounds like we should be using the existing inline grpc_slice size [GRPC_SLICE_INLINED_SIZE](https://source.corp.google.com/piper///depot/google3/third_party/grpc/include/grpc/impl/slice_type.h;bpv=1;bpt=1;l=47?q=grpc_slice&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental&gsn=GRPC_SLICE_INLINED_SIZE&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fgoogle3%3Flang%3Dc%252B%252B%3Fpath%3Dthird_party%2Fgrpc%2Finclude%2Fgrpc%2Fimpl%2Fslice_type.h%23GRPC_SLICE_INLINED_SIZE%2523m%25401561)

Reply:
non-blocking, but please add a TODO to figure this out

Reply:
Done.